<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TestWidget</class>
 <widget class="QWidget" name="TestWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>336</width>
    <height>470</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>模拟测试</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <property name="spacing">
    <number>15</number>
   </property>
   <item>
    <widget class="QWidget" name="widget_3" native="true">
     <property name="styleSheet">
      <string notr="true">        #widget_3 {
            border: 1px solid rgb(110, 100, 100);
        }</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>11</number>
      </property>
      <property name="topMargin">
       <number>11</number>
      </property>
      <property name="rightMargin">
       <number>11</number>
      </property>
      <property name="bottomMargin">
       <number>7</number>
      </property>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <property name="spacing">
         <number>0</number>
        </property>
        <item alignment="Qt::AlignmentFlag::AlignLeft">
         <widget class="QLabel" name="label_5">
          <property name="styleSheet">
           <string notr="true">font: 12pt &quot;微软雅黑&quot;;
color: rgb(0, 212, 255);</string>
          </property>
          <property name="text">
           <string>生成非好友用户</string>
          </property>
         </widget>
        </item>
        <item alignment="Qt::AlignmentFlag::AlignLeft">
         <widget class="QLabel" name="label">
          <property name="styleSheet">
           <string notr="true">font: 9pt &quot;微软雅黑&quot;;
</string>
          </property>
          <property name="text">
           <string>（用于模拟添加好友）</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Orientation::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <property name="spacing">
         <number>11</number>
        </property>
        <item alignment="Qt::AlignmentFlag::AlignLeft">
         <widget class="QSpinBox" name="spinBox">
          <property name="minimumSize">
           <size>
            <width>100</width>
            <height>28</height>
           </size>
          </property>
          <property name="frame">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Orientation::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item alignment="Qt::AlignmentFlag::AlignLeft">
         <widget class="QPushButton" name="pushButton1">
          <property name="minimumSize">
           <size>
            <width>28</width>
            <height>28</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>50</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>确认</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QProgressBar" name="progressBar_1">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>20</height>
         </size>
        </property>
        <property name="value">
         <number>0</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget_2" native="true">
     <property name="styleSheet">
      <string notr="true">        #widget_2 {
            border: 1px solid rgb(100, 100, 100);
        }</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="spacing">
       <number>7</number>
      </property>
      <property name="leftMargin">
       <number>11</number>
      </property>
      <property name="topMargin">
       <number>11</number>
      </property>
      <property name="rightMargin">
       <number>11</number>
      </property>
      <property name="bottomMargin">
       <number>11</number>
      </property>
      <item alignment="Qt::AlignmentFlag::AlignLeft">
       <widget class="QLabel" name="label_2">
        <property name="styleSheet">
         <string notr="true">font: 12pt &quot;微软雅黑&quot;;
color: rgb(0, 212, 255);</string>
        </property>
        <property name="text">
         <string>直接生成好友</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <property name="spacing">
         <number>11</number>
        </property>
        <item>
         <widget class="QSpinBox" name="spinBox_2">
          <property name="minimumSize">
           <size>
            <width>100</width>
            <height>28</height>
           </size>
          </property>
          <property name="frame">
           <bool>false</bool>
          </property>
          <property name="keyboardTracking">
           <bool>true</bool>
          </property>
          <property name="stepType">
           <enum>QAbstractSpinBox::StepType::DefaultStepType</enum>
          </property>
          <property name="displayIntegerBase">
           <number>10</number>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Orientation::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item alignment="Qt::AlignmentFlag::AlignRight">
         <widget class="QPushButton" name="pushButton_2">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>28</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>50</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>确认</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QProgressBar" name="progressBar_2">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>20</height>
         </size>
        </property>
        <property name="value">
         <number>0</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <property name="styleSheet">
      <string notr="true">        #widget {
            border: 1px solid rgb(100, 100, 100);
        }</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_6">
      <property name="spacing">
       <number>7</number>
      </property>
      <property name="leftMargin">
       <number>11</number>
      </property>
      <property name="topMargin">
       <number>11</number>
      </property>
      <property name="rightMargin">
       <number>11</number>
      </property>
      <property name="bottomMargin">
       <number>11</number>
      </property>
      <item>
       <widget class="QLabel" name="label_3">
        <property name="styleSheet">
         <string notr="true">font: 12pt &quot;微软雅黑&quot;;
color: rgb(0, 212, 255);</string>
        </property>
        <property name="text">
         <string>AI接管好友 （用于模拟聊天）</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <property name="spacing">
         <number>8</number>
        </property>
        <property name="bottomMargin">
         <number>9</number>
        </property>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <property name="spacing">
           <number>13</number>
          </property>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <property name="spacing">
             <number>11</number>
            </property>
            <item alignment="Qt::AlignmentFlag::AlignLeft">
             <widget class="QLabel" name="label_4">
              <property name="styleSheet">
               <string notr="true">font: 9pt &quot;微软雅黑&quot;;</string>
              </property>
              <property name="text">
               <string>豆包模型API_Key：</string>
              </property>
             </widget>
            </item>
            <item alignment="Qt::AlignmentFlag::AlignLeft">
             <widget class="QLineEdit" name="lineEdit">
              <property name="styleSheet">
               <string notr="true">QLineEdit {
        border: 1px solid #C0C4CC; /* 默认灰色边框 */
        border-radius: 4px;
        padding: 4px 6px;
        background-color: white; /* 确保背景白色，避免透明问题 */
    }
    QLineEdit:focus {
        border-color: #2D8CF0; /* 聚焦时蓝色边框（可替换为其他颜色） */
        outline: none; /* 清除系统默认聚焦轮廓（可选） */
    }</string>
              </property>
              <property name="frame">
               <bool>false</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_5">
            <property name="spacing">
             <number>11</number>
            </property>
            <item alignment="Qt::AlignmentFlag::AlignLeft">
             <widget class="QLabel" name="label_7">
              <property name="styleSheet">
               <string notr="true">font: 9pt &quot;微软雅黑&quot;;</string>
              </property>
              <property name="text">
               <string>用户Id：</string>
              </property>
             </widget>
            </item>
            <item alignment="Qt::AlignmentFlag::AlignLeft">
             <widget class="QLineEdit" name="lineEdit_2">
              <property name="styleSheet">
               <string notr="true">QLineEdit {
        border: 1px solid #C0C4CC; /* 默认灰色边框 */
        border-radius: 4px;
        padding: 4px 6px;
        background-color: white; /* 确保背景白色，避免透明问题 */
    }
    QLineEdit:focus {
        border-color: #2D8CF0; /* 聚焦时蓝色边框（可替换为其他颜色） */
        outline: none; /* 清除系统默认聚焦轮廓（可选） */
    }</string>
              </property>
              <property name="frame">
               <bool>false</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item alignment="Qt::AlignmentFlag::AlignRight">
         <widget class="QPushButton" name="pushButton_3">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>28</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>50</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>确认</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_6">
          <property name="styleSheet">
           <string notr="true">font: 9pt &quot;微软雅黑&quot;;</string>
          </property>
          <property name="text">
           <string>当前key：</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
